.coveragerc

```diff
@@ -600,6 +600,9 @@ omit =
     homeassistant/components/meteo_france/sensor.py
     homeassistant/components/meteo_france/weather.py
     homeassistant/components/meteoalarm/*
+    homeassistant/components/meteoclimatic/__init__.py
+    homeassistant/components/meteoclimatic/const.py
```

Comment:
We should avoid adding files to .coveragerc, since it means they are not tested. Are you sure const.py and __init__.py are not tested?
you can run coverage locally with:
pytest --cov-report html --cov=../../../homeassistant/components/meteoclimatic .
from your testdirectiry (hence the .)
report is in directory htmlcov

Reply:
That's actually fine. We don't require these to be tested. It's opt-in.